src/events/events.ts

```diff
@@ -315,6 +374,11 @@ export const onUserUpdate = async (client: Client, before: User, after: User): P
  * @param after the message's current state
  */
 export const onMessageEdit = async (client: Client, before: Message, after: Message): Promise<void> => {
+    // Ignore own messages
+    if (before.author.id === client.user.id) {
+        return;
+    }
```

Comment:
The message logs handlers have also been updated to ignore the bot's own messages.